packages/postgres-database/src/simcore_postgres_database/models/projects_to_jobs.py

```diff
@@ -0,0 +1,52 @@
+"""
+    These tables were designed to be controled by projects-plugin in
+    the webserver's service
+"""
+
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .projects import projects
+
+projects_to_jobs = sa.Table(
+    "projects_to_jobs",
+    #
+    # Every job is mapped to a project and has an ancestor (see job_parent_name)
```

Comment:
Question: what is the need of that table? especially if you have a 1:1 relation from job to project?